pallets/common/src/traits/utility.rs

```diff
@@ -0,0 +1,8 @@
+use frame_support::weights::{GetDispatchInfo, Weight};
+
+pub trait WeightInfo {
```

Comment:
Maybe best to keep define this inside the utility pallet rather than in the common pallet?